README.md

```diff
@@ -49,14 +51,18 @@ This sample demonstrates the following MSAL Angular concepts:
 | `tsconfig.*.json` | TypeScript configuration files.            |
 | `tslint.json`     | TS Lint configuration files.               |
 
+**Note:** This sample's structure was generated with the [Angular CLI](https://cli.angular.io/).
+
 ## Prerequisites
 
-[Node.js](https://nodejs.org/en/) must be installed to run this sample.
+- [Node.js](https://nodejs.org/en/) must be installed to run this sample.
+- [Angular CLI](https://cli.angular.io/) must be installed to run this sample.

```

Comment:
The Angular CLI will be installed as a dependency when `npm install` is run, so it is not technically a prerequisitve.

Reply:
removed